webclient/src/components/views/Create.vue

```diff
@@ -30,14 +33,16 @@ export default {
         CurrentIteration: {
           Tasks: []
         },
-        Completed: false
+        Completed: false,
+        loading: false
       },
       taskCount: 1,
       warningDisplayed: false
     };
   },
   methods: {
     SaveProject: function () {
+      var self = this;
```

Comment:
Is this necessary? I think Vue preserves the context so you don't need to do this.